src/components/TopNav.js

```diff
@@ -69,9 +69,10 @@ function TopNav(props) {
 
 function mapStateToProps(state){
     return {
-      messages:state.message
+      messages:state.messages
     }
 }
 
-const ListOfUsersContainer = connect(mapStateToProps)(TopNav);
+const TopNavContainer =
```

Comment:
Think about the logic of what is happening here. Why would you make the variable TopNavContainer if you never use it. How does it need to be used.
You need to go back to the address book project and study the pattern.